share/instruments/instrumentsxml.h

```diff
@@ -189,2502 +195,2918 @@ QT_TRANSLATE_NOOP("InstrumentsXML", "Viols"),
-QT_TRANSLATE_NOOP("InstrumentsXML", "Soprano Sarrusophone"),
-QT_TRANSLATE_NOOP("InstrumentsXML", "Alto Sarrusophone"),
-QT_TRANSLATE_NOOP("InstrumentsXML", "Alto Sarrusophone"),
-QT_TRANSLATE_NOOP3("InstrumentsXML", "A. Sar.", "Alto Sarrusophone"),
-QT_TRANSLATE_NOOP("InstrumentsXML", "Alto Sarrusophone"),
-QT_TRANSLATE_NOOP("InstrumentsXML", "Tenor Sarrusophone"),
-QT_TRANSLATE_NOOP("InstrumentsXML", "Tenor Sarrusophone"),
-QT_TRANSLATE_NOOP3("InstrumentsXML", "T. Sar.", "Tenor Sarrusophone"),
-QT_TRANSLATE_NOOP("InstrumentsXML", "Tenor Sarrusophone"),
-QT_TRANSLATE_NOOP("InstrumentsXML", "Baritone Sarrusophone"),
-QT_TRANSLATE_NOOP("InstrumentsXML", "Baritone Sarrusophone"),
-QT_TRANSLATE_NOOP3("InstrumentsXML", "Bar. Sar.", "Baritone Sarrusophone"),
-QT_TRANSLATE_NOOP("InstrumentsXML", "Baritone Sarrusophone"),
-QT_TRANSLATE_NOOP("InstrumentsXML", "Bass Sarrusophone"),
-QT_TRANSLATE_NOOP("InstrumentsXML", "Bass Sarrusophone"),
-QT_TRANSLATE_NOOP3("InstrumentsXML", "B. Sar.", "Bass Sarrusophone"),
-QT_TRANSLATE_NOOP("InstrumentsXML", "Bass Sarrusophone"),
-QT_TRANSLATE_NOOP("InstrumentsXML", "Contrabass Sarrusophone"),
-QT_TRANSLATE_NOOP("InstrumentsXML", "Contrabass Sarrusophone"),
-QT_TRANSLATE_NOOP3("InstrumentsXML", "Cb. Sar.", "Contrabass Sarrusophone"),
-QT_TRANSLATE_NOOP("InstrumentsXML", "Contrabass Sarrusophone"),
-QT_TRANSLATE_NOOP("InstrumentsXML", "Bagpipe"),
-QT_TRANSLATE_NOOP("InstrumentsXML", "Bagpipe"),
-QT_TRANSLATE_NOOP3("InstrumentsXML", "Bagp.", "Bagpipe"),
-QT_TRANSLATE_NOOP("InstrumentsXML", "Bagpipe"),
+QT_TRANSLATE_NOOP3("InstrumentsXML", "Winds", "winds|longName"),
+QT_TRANSLATE_NOOP3("InstrumentsXML", "Wi.", "winds|shortName"),
+QT_TRANSLATE_NOOP3("InstrumentsXML", "Winds", "winds|trackName"),
+QT_TRANSLATE_NOOP3("InstrumentsXML", "Wind section notated on a grand staff.", "winds|description"),
+QT_TRANSLATE_NOOP3("InstrumentsXML", "Piccolo", "eb-piccolo|longName"),
+QT_TRANSLATE_NOOP3("InstrumentsXML", "Picc.", "eb-piccolo|shortName"),
+QT_TRANSLATE_NOOP3("InstrumentsXML", "Piccolo", "eb-piccolo|trackName"),
+//: traitName
+QT_TRANSLATE_NOOP("InstrumentsXML", "E♭"),
+QT_TRANSLATE_NOOP3("InstrumentsXML", "Piccolo in E♭, sounding a minor third above the standard piccolo.", "eb-piccolo|description"),
+QT_TRANSLATE_NOOP3("InstrumentsXML", "Piccolo", "db-piccolo|longName"),
+QT_TRANSLATE_NOOP3("InstrumentsXML", "Picc.", "db-piccolo|shortName"),
+QT_TRANSLATE_NOOP3("InstrumentsXML", "Piccolo", "db-piccolo|trackName"),
+//: traitName
+QT_TRANSLATE_NOOP("InstrumentsXML", "D♭"),
+QT_TRANSLATE_NOOP3("InstrumentsXML", "Piccolo in D♭, sounding a semitone above the standard piccolo.", "db-piccolo|description"),
+QT_TRANSLATE_NOOP3("InstrumentsXML", "Piccolo", "piccolo|longName"),
+QT_TRANSLATE_NOOP3("InstrumentsXML", "Picc.", "piccolo|shortName"),
+QT_TRANSLATE_NOOP3("InstrumentsXML", "Piccolo", "piccolo|trackName"),
+//: traitName
+QT_TRANSLATE_NOOP("InstrumentsXML", "*(C)"),
```

Comment:
It was clever to use a translation comment like this but I think we will need disambiguation text even for the trait names. The reason is that we add special characters to the trait names to give them special meanings:

`*` - This is the most common trait name so use it as the default in the New Score Wizard dropdown.
`()` - This trait name should not appear in the score.

These are added to the translatable string to allow translators to choose a different default to use in their locale, and to hide or unhide the trait name in scores written in their language.

If the `//:` comment strings are actually exposed to translators then you could make use of it like this:

```c++
//: traitName codes: * most common for instrument; () hide in score
QT_TRANSLATE_NOOP3("InstrumentsXML", "*(C)", "piccolo|traitName"),
```

A translator could choose to translate `"*(C)"` for the piccolo as `"C"`, `"*C"`, `"(C)"` or `"*(C)"`, and they might make a different choice for a different instrument that also has `"*(C)"` as the trait name in English, hence we need disambiguation text.